cocos/index.html

```diff
@@ -1,966 +1,799 @@
-                  >Twitter</a
-                >
-              </li>
-              <li>
-                <i class="fab fa-linkedin me-2"></i
-                ><a
-                  href="https://www.linkedin.com/company/ultravioletrs"
-                  class="text-white"
-                  target="_blank"
-                  >LinkedIn</a
-                >
-              </li>
-              <li>
-                <i class="fab fa-medium me-2"></i
-                ><a
-                  href="https://medium.com/ultraviolet-blog"
-                  class="text-white"
-                  target="_blank"
-                  >Medium Blog</a
-                >
-              </li>
-              <li>
-                <i class="fab fa-github me-2"></i
-                ><a
-                  href="https://github.com/ultravioletrs"
-                  class="text-white"
-                  target="_blank"
-                  >GitHub</a
-                >
-              </li>
-            </ul>
-          </div>
-          <div class="col">
-            <h5>Contact Us</h5>
-            <p>
-              <i class="fas fa-envelope me-2"></i
-              ><a href="mailto:info@ultraviolet.rs" class="text-white"
-                >info@ultraviolet.rs</a
-              >
-            </p>
-            <p>
-              <i class="fab fa-gitter me-2"></i
-              ><a
-                href="https://app.gitter.im/#/room/#Ultraviolet_community:gitter.im"
-                class="text-white"
-                target="_blank"
-                >Chat on Gitter</a
-              >
-            </p>
-            <p>
-              <i class="fas fa-map-marker-alt me-2"></i>Bulevar Arsenija
-              Carnojevica 103, 11000 Belgrade, Serbia
-            </p>
-          </div>
+    </div>
+  </section>
+  <!-- End of Newsletter section -->
+
+  <!-- Footer -->
+  <footer class="bg-dark text-white py-3">
+    <div class="container">
+      <div class="row mt-4 mb-3">
+        <div class="col">
+          <h5>About Us</h5>
+          <p>
+            Ultraviolet is a leading company specializing in confidential
+            computing, cloud security, AI/ML, multi-party computation, and
+            secure data sharing.
+          </p>
+        </div>
+        <div class="col">
+          <h5>Products</h5>
+          <ul class="list-unstyled">
+            <li>
+              <i class="fas fa-network-wired me-2"></i><a href="prism/" class="text-white">Prism AI</a>
+            </li>
+            <li>
+              <i class="fas fa-cog me-2"></i><a href="cocos/" class="text-white">Cocos AI</a>
+            </li>
+            <li>
+              <i class="fas fa-cube"></i>
+              <a href="cube/" class="text-white">Cube AI</a>
+            </li>
+          </ul>
+          <h5>Resources</h5>
+          <ul class="list-unstyled">
+            <li>
+              <a href="https://docs.google.com/presentation/d/1oZDjQjuzNR8PeI1AHS8tePaiA73SmjnjrFFEBoS6b68/export/pdf"
```

Comment:
Let's use our CDN for this instead: https://cdn.ultraviolet.rs/datasheets/Cocos%20AI%20-%20Presentation.pdf